### DISCUSSION TITLE
Jupyter Notebook can't display the chart

### DISCUSSION THREAD

**Discussion by 893239571** [vega/altair#2611] [Q&A] [answered]
I can not display normally in jupyter notebook. Although i use alt.renderers.enable('notebook).There some problems in browser console:

Uncaught Error loading script: Script error for "vega-embed"
Failed to load resource: net::ERR_CONNECTION_RESET
Failed to load resource: the server responded with a status of 404 (Not Found)

How to solve it?

**Comment by joelostblom** [comment, marked as answer]
Could you try if the same piece of code works fine for you in JupyterLab with the default renderer as well as making sure you are on the latest version of the notebook and Altair?

**Reply by 893239571** [reply]
I am sure that the version of the notebook and Altair are latest. I also try in JupyterLab, it failed.

This is in JupyterLab
![2022-06-06 10-02-44屏幕截图](https://user-images.githubusercontent.com/58102171/172082781-df755a68-02c5-4dab-9f17-a5fcfde518fa.png)

This is in Jupyter Notebook
![2022-06-06 10-04-31屏幕截图](https://user-images.githubusercontent.com/58102171/172082780-da1ddc86-29e3-4bd3-afa2-34595728a2f6.png)

**Reply by joelostblom** [reply]
Hmm I am not sure what the exact cause is, but what I would try next is a different browser, disabling all addons, checking if there is something in your firewall that might block the connection to the vega script, and trying from a different internet connection. Related issues are https://github.com/vega/ipyvega/issues/412 and https://github.com/altair-viz/altair/issues/2594

**Reply by 893239571** [reply]
Thank you for your answer. I have found the solution. I can not visit the CDN. Therefore i change the other CDN. It works!
